docs/whats-new/2021-02.md

```diff
@@ -145,7 +145,7 @@ The following people contributed to the .NET docs during this period. Thank you!
 - [nschonni](https://github.com/nschonni) - Nick Schonning (1)
 - [parelkobra](https://github.com/parelkobra) - David Ruiz (1)
 - [pavel-mikula-sonarsource](https://github.com/pavel-mikula-sonarsource) - Pavel Mikula (1)
-- [realcooltrev](https://github.com/realcooltrev) - Trevor Pierce (1)
+- [realcooltrev](https://github.com/ketchup-cfg) - Trevor Pierce (1)
```

Comment:
nit: link text should have been renamed, but no idea if it's worth doing that in a new PR.